src/spyglass/position/v1/position_dlc_pose_estimation.py

```diff
@@ -232,25 +232,38 @@ def make(self, key):
                 dlc_result.creation_time
             ).strftime("%Y-%m-%d %H:%M:%S")
 
-            logger.logger.info("getting raw position")
-            interval_list_name = (
-                convert_epoch_interval_name_to_position_interval_name(
-                    {
-                        "nwb_file_name": key["nwb_file_name"],
-                        "epoch": key["epoch"],
-                    },
-                    populate_missing=False,
+            # get video information
+            _, _, meters_per_pixel, video_time = get_video_path(key)
+            # check if a position interval exists for this epoch
+            try:
+                interval_list_name = (
+                    convert_epoch_interval_name_to_position_interval_name(
```

Comment:
It's not clear to me why `convert_epoch...` should only raise an error in the `populate_missing` case. I think  might add another arg for `raise_err` over there to avoid try/except. I was taught that try/except only belongs in production if you can't modify the underlying behavior, eg on outside package

Reply:
@ CBroz1 agreed. I'll tweak that function